plugins/MultitapEcho/MultitapEcho.cpp

```diff
@@ -118,8 +118,8 @@ bool MultitapEchoEffect::processAudioBuffer( SampleFrame* buf, const fpp_t frame
 	}
 	
 	// add dry buffer - never swap inputs for dry
-	m_buffer.writeAddingMultiplied( buf, 0, frames, dryGain );
-	
+	m_buffer.writeAddingMultiplied(buf, static_cast<size_t>(0), frames, dryGain);
```

Comment:
Why do we need to do a static cast here? Wouldn't it be better to change the calling function, assuming all callers are changed.